.circleci/config.yml

```
@@ -1,7 +1,135 @@
 version: 2.1
 orbs:
-  node: circleci/node@3.0.0
+  node: circleci/node@1.1.6
+jobs:
+  install-prod:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.1
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+              - v1-dependencies-
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+  build-prod:
```

let's make this the convention
```suggestion
  prod-build:
```